events/importer/osterbotten.py

```diff
@@ -83,12 +85,19 @@ def import_events(self):
         keyword_matcher = KeywordMatcher()
         for lang, locale in OSTERBOTTEN_PARAMS['languages'].items():
             start = 0
-            while (len( self.items_from_url( self.getUrl(locale, start) ) ) > 0):
+            loop = True
+            while (loop):
                 items = self.items_from_url( self.getUrl(locale, start) )
-                start = start + 10
-                for item in items:
-                    self._import_event(lang, item, events, keyword_matcher)
-                organizers = self._import_organizers_from_events(events)
+
+                if items and len(items) > 0:  
+                    start = start + 10
+                    for item in items:
+                        self._import_event(lang, item, events, keyword_matcher)
+                    organizers = self._import_organizers_from_events(events)
+                elif items == None:
```

Comment:
Use `items is None` instead of `items == None`, the former is more idiomatic, and protects from weird `==` implementations.